.github/workflows/node.js.yml

```
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master, release/*, chore/* ]
+  pull_request:
+    branches: [ master , release/*]
```

Let's add `develop` as well and fix the conventions in this repo.
```suggestion
    branches: [ master , develop , release/*]
```